scripts/generate-java-code-coverage.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+JACOCO_VERSION=0.8.12
+
+if [ "$(basename "$(pwd)")" != "java" ]; then
+    echo "This script must be run from the 'java' directory."
+    exit 1
+fi
+
+# make -C ../cpp srcs
+# make
```

Comment:
We need these uncommented to actually run the build, right?

Reply:
Yes! Fixed.